### DISCUSSION TITLE
Tailwind pixel unit override not working?

### DISCUSSION THREAD

**Discussion by mbforbes** [resend/react-email#1511] [Q&A] [answered]
This section of the docs helpfully describes how to override Tailwind's config to use `px` instead of `rem`, since `rem` isn't universally supported by email providers:
https://react.email/docs/components/tailwind#tailwind-configuration-with-px-instead-of-rem

When I create a `tailwind.config.ts` file with those contents, my intellisense picks it up and correctly shows the overrides:

![Screenshot 2024-06-20 at 3 36 45 PM](https://github.com/resend/react-email/assets/1170062/b2311f83-5986-44f7-9148-118bb190468e)
_Before_

![Screenshot 2024-06-20 at 3 35 53 PM](https://github.com/resend/react-email/assets/1170062/48c1510c-a064-4b58-abc6-0313a78d0345)
_After_

However, when I build an export like

```sh
yarn run email export --outDir ../backend/data/emails/ --pretty
```

... all the embedded tailwind units are still `rem`. Is this a bug? Or do I need to put my config somewhere else, or point something to it? (Same thing happens if I remove types and name it `tailwind.config.js`.)

Thank you for any help!

**Comment by gabrielmfern** [comment, marked as answer]
That's because our Tailwind is currently a little unconventional, in that you need to specify the config as a prop to `<Tailwind>`. So what you need to do is something like this:

```tsx
// emails/my-email-template.tsx

// ... other imports
import TailwindConfig from '../tailwind.config';

export default Email() {
  return <Tailwind config={TailwindConfig}>
    {/* ... */}
  </Tailwind>;
}
```

After that it should be working as you expect.

**Reply by mbforbes** [reply]
Awesome, thank you!

It fixed everything except `border-radius`.

Did I miss this in the docs? Might it be possible to add this snippet to that docs location I linked?
